src/lib/detectInstall.js

```diff
@@ -38,6 +38,12 @@ function detectInstall (context) {
   let packageName = plugin.split('@')[0]
   let packageVersion = plugin.split('@')[1] || null
 
+  let global = false
```

Comment:
We won't want to use `global`, since it [conflicts with a predefined Node.js variable](https://nodejs.org/api/globals.html#globals_global). Consider using `isGlobal` instead.